bin/liquidator/src/liquidate.rs

```diff
@@ -541,9 +545,14 @@ pub async fn maybe_liquidate_account(
     let liqor_min_health_ratio = I80F48::from_num(config.min_health_ratio);
 
     let account = account_fetcher.fetch_mango_account(pubkey)?;
-    let health_cache = health_cache::new(&mango_client.context, account_fetcher, &account)
-        .await
-        .context("creating health cache 1")?;
+    let health_cache = health_cache::new(
+        &mango_client.context,
+        &mango_client.client.fallback_oracle_config,
+        account_fetcher,
+        &account,
```

Comment:
hmm, wonder if there could be a mango_client helper function to get the health cache for an account -- after all the context, fallback_oracle_config and account_fetcher are all available to the mango_client